enhancements/network/static-from-dhcp.md

```diff
@@ -0,0 +1,152 @@
+---
+title: static-ip-addresses-from-dhcp
+authors:
+  - "@cybertron"
+reviewers:
+  - TBD
+  - "@celebdor"
+  - "@bcrochet"
+  - "@yboaron"
+approvers:
+  - TBD
+creation-date: 2020-10-28
+last-updated: 2020-10-29
+status: implementable
+---
+
+# Static IP Addresses from DHCP
+
+## Release Signoff Checklist
+
+- [X] Enhancement is `implementable`
+- [X] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+In some OpenShift deployments, particularly Edge deployments, DHCP may not
```

Comment:
Questions:
1. Is this relevant to VIPs address assignment by DHCP, or just to node address assignment?
2. In case of installation by assisted-installer, what is the suggestion to handle DHCP leases before and after the installation has started?  Does it imply changes in assisted installer that performs DHCP VIP addresses allocation?

Reply:
This is largely separate from the DHCP VIPs. The only implications it might have is that we need to make sure if the VIPs were assigned by DHCP they handle losing access to the DHCP server. Probably that means making sure the lease file we use for dhclient is in a persistent location, if it isn't already.

Otherwise this doesn't change much in terms of installation. A DHCP server is still required for initial installation to provide the IPs in the first place.